scripts/run.e2e.sh

```diff
@@ -56,7 +56,17 @@ then
 	extra_build_args="-tags ledger_zemu"
 fi
 
-ACK_GINKGO_RC=true ginkgo build $extra_build_args ./tests/e2e
+ACK_GINKGO_RC=true ~/go/bin/ginkgo build $extra_build_args ./tests/e2e
+
+if [ "${COVERAGE_MODE:-}" == true ]
+then
+	echo 'Coverage mode enabled - re-creating coverage dir'
+	echo 'It requires the CLI binary to be built by build.sh with COVERAGE_MODE=true too'
+	export GOCOVERDIR=coverage
```

Comment:
can you set it abs path?

Reply:
and fixed location say, inside repo

Reply:
this script is supposed to be run by CI with another script to be created next, in which pwd defaults to repo root. i can get to it when needed.